requirements.txt

```diff
@@ -1 +1,3 @@
-mig-dx-api==0.1.5
+mig-dx-api==0.1.6
```

Comment:
I don't have strong feelings here, but might be nice to alphabetize as dependencies grow.